third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -55,13 +55,13 @@ PjRtComputationClient::PjRtComputationClient() {
     bool async = sys_util::GetEnvBool(env::kEnvPjrtAsyncCpuClient, true);
     int cpu_device_count = sys_util::GetEnvInt(env::kEnvNumCpu, 1);
     client_ = std::move(xla::GetTfrtCpuClient(async, cpu_device_count).value());
-  } else if (device_type == "TPU") {
-    TF_VLOG(1) << "Initializing PjRt TPU client...";
+  } else if (device_type == "TPU_LEGACY") {
+    TF_VLOG(1) << "Initializing PjRt StreamExecutor TPU client...";
     int64_t max_inflight_computations = sys_util::GetEnvInt(
         env::kEnvPjRtTpuMaxInflightComputations, /*defval=*/32);
     client_ = xla::GetTpuClient(max_inflight_computations).value();
-  } else if (device_type == "TPU_C_API") {
-    TF_VLOG(1) << "Initializing PjRt C API client...";
+  } else if (device_type == "TPU" || device_type == "TPU_C_API") {
```

Comment:
very minor nit, can we move this to the top of the if check? I think it is easier to read if we put the most important/common case in the first block of if check.

Reply:
Good point. Done